lstchain/tools/lstchain_create_drs4_pedestal_file.py

```diff
@@ -257,6 +259,65 @@ def mean_spike_height(self):
 
         return spike_heights
 
+    def check_baseline_values(self, baseline_mean):
+        """
+        Check the values of the drs4 baseline for issues.
+
+        In case of no EVBv5 data or no pre-calibration by EVBv6,
+        we expect no negative and no small values.
+
+        In case of EVBv6 applied baseline corrections, values should
+        be close to 0, but negative values are ok.
+        """
+
+        check_large_values = False
+        check_negative = True
+        check_small = True
+        
+        if self.source.cta_r1:
+            preprocessing = self.source.evb_preprocessing[TriggerBits.MONO]
+            if EVBPreprocessing.BASELINE_SUBTRACTION in preprocessing:
+                check_large_values = True
+                check_negative = False
+                check_small = False
+
+
+        if check_negative:
+            negative = baseline_mean < 0
+            n_negative = np.count_nonzero(negative)
+            if n_negative > 0:
+                gain, pixel, capacitor = np.nonzero(negative)
+                self.log.critical(f'{n_negative} baseline values are smaller than 0')
+                self.log.info("Gain | Pixel | Capacitor | Baseline ")
+                for g, p, c in zip(gain, pixel, capacitor):
+                    self.log.info(
+                        f"{g:4d} | {p:4d} | {c:9d} | {baseline_mean[g][p][c]:6.1f}"
+                    )
+
+        if check_small:
```

Comment:
To keep the same behaviour, shouldn't we have a check for large negative values in the case of baseline-subtracted waveforms?  I don't know though if that covers a possible error, or is a useless check.

Reply:
Tests not passing, but they should still work, right?
Message is : "Input pedestal must have dtype (u)int16, got uint16"

Reply:
> Tests not passing, but they should still work, right?

I failed logic...